lib/engine/game/g_18_india/step/home_track.rb

```diff
@@ -43,12 +43,38 @@ def process_lay_tile(action)
               connected: false,
               extra_action: true
             )
+
             @round.pending_tokens.shift
           end
 
+          def place_second_oo_token(tile, corp_name)
+            corporation = @game.corporation_by_id(corp_name)
+            return unless corporation&.floated
+
+            token = corporation.next_token
+            city = tile.cities.reject(&:tokened?).first
+            city.place_token(corporation, token) if city.tokenable?(corporation, tokens: token)
+          end
+
           def process_place_token(action)
             super
             tile = action.city.tile
+            # apply "other" OO token once first to operate is applied if floated
+            if action.entity.name == 'NWR' && tile.name == '235'
+              place_second_oo_token(tile, 'SPD')
+              @round.pending_tokens.shift
+            elsif action.entity.name == 'SPD' && tile.name == '235'
+              # remove and swap the NWR that appeared... hacky...
+              city = action.city
+              old_token = city.tokens.first
+              old_token.remove!
+              city.exchange_token(action.entity.find_token_by_type)
+
+              # apply "other" OO token once first to operate is applied if floated
+              place_second_oo_token(tile, 'NWR')
+              @round.pending_tokens.shift
+            end
```

Comment:
You mentioned elsewhere some other companies/tiles that could also be impacted by this. It got me thinking, and you might be able to use [`case/when`](https://docs.ruby-lang.org/en/3.2/syntax/control_expressions_rdoc.html#label-case+Expression) (Ruby's `switch`) to DRY it up a little bit and make it simpler to generalize to add those other companies and tiles, for example:

```suggestion
            other_corp =
              case [action.entity.name, tile.name]
              when ['NWR', '235']
                'SPD'
              when ['SPD', '235']
                # remove and swap the NWR that appeared... hacky...
                city = action.city
                old_token = city.tokens.first
                old_token.remove!
                city.exchange_token(action.entity.find_token_by_type)
                'NWR'
              end
            if other_corp
              # apply "other" OO token once first to operate is applied if floated
              place_second_oo_token(tile, other_corp)
              @round.pending_tokens.shift
            end
```